README.md

```diff
@@ -56,7 +55,49 @@ In a nutshell, a few EPICs:
 - Implement the first stage: "Project Assemble" instead of relying on a pre-built, pre-configured image.
 - Review the API interfaces and types to make sure that aligns with the build abstraction and can cover the majority of use cases.
 
-Keep in mind that the end goal is to use this package in any environment that can run a Go application, despite being a cluster or a local environment.
+Keep in mind that the end goal is to use this package anywhere you can run a Go application.
 
-As we evolve, evaluate the package with Camel-K team to make sure that it can fit their use case the same way it does today with their embedded package. 
+As we evolve, evaluate the package with Camel-K team to make sure that it can fit their use case the same way it does today with their embedded package.
 That might require some work to remove the relation of the integration concept from the build tasks.
+
+## Docker Registry configuration
+
+If we want to connect on a remote server using Docker we must set as environment variables the follows:
+
+- **DOCKER_HOST**: sets the url to the docker server.
+- **DOCKER_API_VERSION**: sets the version of the API to reach, leave empty for latest.
+- **DOCKER_CERT_PATH**: loads the TLS certificates from.
+- **DOCKER_TLS_VERIFY**: enables or disable TLS verification, off by default.
+
+otherwise a local docker registry will be used if nothing is present
+
+`docker container ls -la`
+
+
+## Podman Registry configuration
+
+If we want to connect on a remote server using Podman we can use one of the following uri connections, tcp, unix socket or ssh
+
+- tcp://localhost:<port>
+- unix:///run/podman/podman.sock
+- ssh://<user>@<host>[:port]/run/podman/podman.sock?secure=True
+
+To connect with a remote server we must set as environment variables the follows:
```

Comment:
user must set ssh one even if he choose the socket option, for example?

Reply:
No, this are the option available